src/GafferSceneUIModule/LightEditorBinding.cpp

```diff
@@ -274,12 +274,24 @@ class MuteColumn : public InspectorColumn
 			}
 			else
 			{
-				auto fullAttributes = scenePath->getScene()->fullAttributes( scenePath->names() );
-				if( auto fullValue = fullAttributes->member<BoolData>( "light:mute" ) )
+				ScenePlug::PathScope pathScope( Context::current() );
```

Comment:
This needs to be `scenePath->context()` rather than `Context::current()`. PathListingWidget doesn't know anything about contexts, so the current context is just going to be a default one.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4948/commits/284acdd56ca77dc09f2ce3d06122cde6420b7656